agent/src/agent/pipeline/pipeline.py

```diff
@@ -283,6 +291,11 @@ def get_tags(self) -> dict:
         }
 
 
+class TestPipeline(Pipeline):
```

Comment:
what this class does?

Reply:
we choose an appropriate config loader depending on a Pipeline type, here https://github.com/anodot/daria/blob/1d5b8cf547491544ee319830fa951928fd24bacd/agent/src/agent/streamsets/manager.py#L109